types/node/test/child_process.ts

```diff
@@ -59,6 +59,20 @@ import { Writable, Readable, Pipe } from 'stream';
     ipc.ref();
 }
 
+{
+    const forked = childProcess.fork('./', {
+        windowsVerbatimArguments: true,
+        silent: false,
+        stdio: ["inherit"],
```

Comment:
A bit inconsistent double quote and single quote. But anyway, we don't have strict rule for this.